CHANGELOG.md

```diff
@@ -9,6 +9,14 @@ and this project adheres to
 
 ## Unreleased
 
+### Added
+
+- Improved type definitions to allow for expressing the type of the
+  `IntegrationExecutionConfig` that is returned from `loadExecutionConfig` and
+  passed to `executionHandler` functions.
+- Added `IntegrationStepExecutionContext.stepMetadata` to allow
```

Comment:
What use cases do you imagine needing the stepMetadata?

Reply:
The integration I'm working on for generating load on persister/core is going to use step metadata in the `executionHandler` bound to the step to generate data. After getting some sleep, I realized I can easily reference the step metadata object because it is a module constant in the file where the `executionHandler` function is defined. It seems natural that the `IntegrationStepExecutionContext` would have the metadata on it, so I carried on with making it work.